content/en/docs/concepts/configuration/configmap.md

```diff
@@ -0,0 +1,78 @@
+---
+title: ConfigMaps
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+{{< glossary_definition term_id="configmap" prepend="A ConfigMap is" length="all" >}}
+
+{{< caution >}}
+ConfigMap does not provide secrecy or encryption.
+
+If the data you want to store are confidential, use a
+{{< glossary_tooltip text="Secret" term_id="secret" >}} rather than a ConfigMap,
+or use additional (third party) tools to keep your data private.
+{{< /caution >}}
+
+{{% /capture %}}
+
+{{% capture body %}}
+## ConfigMap object
+
+A ConfigMap is an API [object](/docs/concepts/overview/working-with-objects/kubernetes-objects/)
+that lets you store configuration for other objects to use. Unlike most
+Kubernetes objects that have a `spec`, a ConfigMap has a `data` section where you
+can place items (keys) and their values.
+
+## ConfigMaps and Pods
+There are three different ways that you can use a ConfigMap to configure
+a container inside a Pod:
+
+- Command line arguments to the entrypoint of a container
+- Environment variables for a container
+- Add a file in read-only volume, for the application to read
+
+These different methods lend themselves to different ways of modeling
```

Comment:
> These different methods lend themselves to different ways of modeling the data being consumed.

I'm not sure what you're trying to get at here – I think the subject, "the data being consumed" is confusing to me.